pkg/virt-launcher/virtwrap/util/libvirt_helper.go

```diff
@@ -269,6 +297,9 @@ func startVirtlogdLogging(stopChan chan struct{}, domainName string) {
 
 		go func() {
 			logfile := fmt.Sprintf("/var/log/libvirt/qemu/%s.log", domainName)
+			if nonRoot {
+				logfile = filepath.Join("/var", "run", "libvirt", "qemu", "log", fmt.Sprintf("%s.log", domainName))
```

Comment:
can we make this the default log location for both root and non root?

Reply:
I am not aware of settings that would allow this.

Reply:
maybe the `log_outputs` setting in libvirtd.conf https://libvirt.org/kbase/debuglogs.html#persistent-setting 

If this isn't trivial, it's not worth it. i'm just trying to find ways to converge the root vs nonroot code paths.

Reply:
That's only for libvirtd, afaik. Domain logs are not affected. 
We already do this for libvirtd https://github.com/kubevirt/kubevirt/blob/master/cmd/virt-launcher/libvirtd.conf .

Reply:
alrighty, no big deal :) thanks for looking